### DISCUSSION TITLE
"Microsoft Stream" - any way to download?

### DISCUSSION THREAD

**Discussion by shellig** [aclap-dev/video-downloadhelper#542] [Help and Q&A]
### Your question

"Microsoft Stream" - any way to download? I see chunks of the video available (and lots of them) is there some setting or something that I can do to get the whole video?
Prior to the new update, it did sometimes load a "chunk"  but I just tried and got: (I took out company and employee identifiers - also shortened the "input file" URL.
"Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

SideDownload error: : [in#0 @ 0000012339aeac40] Error opening input: Invalid data found when processing input
Error opening input file https://southcentralus1-mediap.svc.ms/transform/videotranscode/ad549923fbbc40c072fb5abe991fccc8d14ee0ac09855b782f8d15b8f0d13b79?provider=Spo&docId=https%3A%2F%2FCOMPANYNAME-my.sharepoint.com%2Fpersonal%2FWORKONEDRIVEUSER....
Error opening input files: Invalid data found when processing input"

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"c1a13fa0a4dd4538"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\godfreys\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

The usual method of diagnosing a problem is for somebody else to visit the same web page.  If you can't permit that for reasons of corporate security or something similar, it's going to be pretty much impossible to help you.
